### DISCUSSION TITLE
How to follow people from other misskey instances?

### DISCUSSION THREAD

**Discussion by orzklv** [misskey-dev/misskey#11335] [Q&A]
I'm quite new on activitypub thingies and I've been struggling to find out whether how to follow people from other misskey instances.

**Comment by orzklv** [comment]
I've actually hosted my own instance at misskey.uz and I tried to search people, but it fails to find one displaying this:
![image](https://github.com/misskey-dev/misskey/assets/54666588/cc918cb9-e8bd-4da8-b4bc-d32046422132)

**Reply by tassoman** [reply]
You should use remote search tool, you probably can't find account there because not yet federated.
Go in "more" menu, and find the magnifier lens on the world 🌐🔍 

![immagine](https://github.com/misskey-dev/misskey/assets/92727/3ad4e721-f691-414f-a8b4-e66e2291dfb9)

**Reply by SeleDreams** [reply]
Another option is to add a relay server I think. But most relay servers I tried stay pending forever lol
